### ISSUE TITLE
V2: "not exist, faking successful response"

### ISSUE THREAD

**Issue by IvRRimum** [volatiletech/authboss#177]
Not too big of an issue, but i think we all agree that this shouldn't be there :D

recover/recover.go
```
93    logger.Infof("user %s was attempted to be recovered, user does not exist, faking successful response", recoverVals.GetPID())
```

**Comment by aarondl** [comment]
So there's no password leaks going on here. Are we worried about divulging too much information by including username/e-mail or what have you? I saw this as useful logging from a support piece. And the functionality of faking a successful response is very useful since we never want to divulge if an attacker was right in guessing an e-mail. These kinds of logs can also lead to the discovery of foul-play and bruteforce attempts. I'm definitely keen on improving it in whatever way we need to, but I'm gonna need to be thrown the obvious bone here. It'd also be good to see some best practice documentation to support whatever we're going to do here.
